### PR TITLE
[ACTION] Notion - Query a database #2570

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/notion/actions/query-database/query-database.mjs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    databaseId: {
+      propDefinition: [
+        notion,
+        "databaseId",
+      ],
+    },
+    filter: {
+      label: "Filter",
+      description: "The filter to query. [See how to filters works here](https://developers.notion.com/reference/post-database-query-filter)",
+      type: "string",
```
Comment (vellames-turing) [inline components/notion/actions/query-database/query-database.mjs:20]: Can this be an object?

**Review by vellames-turing — CHANGES_REQUESTED** [review]
Hi there! Just some suggestions

**Review by vellames-turing — APPROVED** [review]
THank you!

**Comment by DilanAthukorala** [comment]
@lcaresia Observing the following issue, could you please check and define how the filter query should be constructed in the UI.

image

**Comment by lcaresia** [comment]
@DilanAthukorala Exampel added!

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Notion_2570.pdf](https://github.com/PipedreamHQ/pipedream/files/8688385/Notion_2570.pdf)

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Notion_2570_0017.pdf](https://github.com/PipedreamHQ/pipedream/files/8689678/Notion_2570_0017.pdf)

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**